syntax = "proto3";
package TFGProtocol;
option csharp_namespace = "TFG.Protobuf";

message DadaGenerada 
{
	double valor=1;
	string nomSensor=2;
}

message CalculDada
{
	string variableCalcular=1;
	string variableRebuda=2;
	double valorRebut=3;	
	uint32 timestampRebut=4;
	uint32 timestampActual=5;
}

message DadaCalculada
{
	string nomVariable = 1;
	double valor = 2;
	uint32 timestamp = 3;
}

message Dada
{
	string nomVariable = 1;
	double valor=2;	
}

message ConsultaDades
{
	repeated string nomVariable = 1;
}

message RespostaDades
{
	repeated Dada Dades=1;
}



